test/setup.ts

```diff
@@ -10,24 +12,39 @@ import {
 	populateIpRangeList,
 	populateNockCitiesList,
 } from './utils/populate-static-files.js';
-
 import chaiOas from './plugins/oas/index.js';
 import { getRedisClient, initRedis } from '../src/lib/redis/client.js';
-import { client } from '../src/lib/sql/client.js';
-import { USERS_TABLE } from '../src/lib/adopted-probes.js';
+import { client as sql } from '../src/lib/sql/client.js';
+
+if (process.env['NODE_ENV'] !== 'test') {
+	throw new Error(`NODE_ENV is not 'test' but '${process.env['NODE_ENV']}'.`);
+}
```

Comment:
Let's keep the check as it was in the other repo. Env itself doesn't mean much.

```js
if (!dbConfig.connection.database.endsWith('-test') && dbConfig.connection.host !== 'localhost') {
	throw new Error(`Database name for test env needs to end with "-test" or the host must be "localhost". Got "${dbConfig.connection.database}"@"${dbConfig.connection.host}".`);
}
```

Reply:
Do you want to divide `connection: 'mysql://directus:password@localhost:3306/directus',` on separate fields in config, or check for `connection !== 'mysql://directus:password@localhost:3306/directus'` in setup.ts?